### PR TITLE
fix(OverlayToolbar): align close icon right

### PR DESCRIPTION
**Pull request by annatraussnig**
Make OverlayToolbarClose consistent with our actual layout and conventions

### CODE REVIEW

**Comment by tpreusse** [comment]
Doc page looks broken now:
Screenshot 2021-07-05 at 14 44 50


Maybe we should also document how to do titles for the most common use case:
Screenshot 2021-07-05 at 14 32 21

A nice API could be:
```js
<OverlayToolbar title='Teilen' onClose={onCloseFn} />
```

`title` being a `PropTypes.node` – also supporting custom elements:
```js
<OverlayToolbar title={<><MyTabButton>Tab 1</MyTabButton> <MyTabButton>Tab 2</MyTabButton></>} onClose={onCloseFn} />
```

**Comment by annatraussnig** [comment]
that's an idea. i've got a bit of time today, i can clean up the overlay api.

**Comment by annatraussnig** [comment]
ready for rereview

File: src/components/Overlay/OverlayToolbar.js [hunk 1]
```diff
@@ -47,41 +52,31 @@ const styles = {
...
+    color: '#fff',
+    pointerEvents: 'none',
+    position: 'absolute',
+    right: 60,
+    top: 9
   })
 }
 
-export const OverlayToolbar = ({ children }) => {
+const Tag = ({ text }) => {
```
Comment (tpreusse) [inline src/components/Overlay/OverlayToolbar.js:70]: Is this being used? Maybe I'm missing something or an artefact from trying things out?

**Review by tpreusse — APPROVED** [review]
Awesome, it's so simple now. Thx

File: src/components/Overlay/OverlayToolbar.js (hunk 1 above)
Comment (annatraussnig) [reply src/components/Overlay/OverlayToolbar.js:70]: precisely. i remove

**Pull request merged**